api/onnx_web/chain/tile.py

```diff
@@ -1,11 +1,14 @@
 from logging import getLogger
 from math import ceil
 from typing import List, Optional, Protocol, Tuple
+from enum import Enum
+import itertools
 
 import numpy as np
 from PIL import Image
 
 from ..params import Size, TileOrder
+from ..image.noise_source import noise_source_histogram
```

Comment:
Please run `make lint-fix` to sort these and fix any other lint issues: https://git.apextoaster.com/ssube/onnx-web/-/jobs/488311